### PR TITLE
The attempts to diagnose asay/msay dropouts

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Redis is randomly dropping on the new server. This is gonna take a bit to diagnose, so let the pain begin.

The PR is now for merge as I added actual functionality 

## Why It's Good For The Game
Stuff should work

## Testing
I compiled it

## Changelog
:cl: AffectedArc07
fix: Fixed redis error reporting 
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/redis.dm [hunk 1]
```diff
@@ -75,7 +85,15 @@ SUBSYSTEM_DEF(redis)
...
-			message_admins("Redis error: [usable_data[channel]] | Please inform the server host.") // uh oh
+			var/redis_error_data = usable_data[channel]
+			var/error_str
+			if(islist(redis_error_data))
+				error_str = json_encode(redis_error_data)
+			else
+				error_str = redis_error_data
+
+			message_admins("Redis error: [error_str] | Please inform the server host.") // uh oh
+			log_game("Redis error: [error_str]")
```
Comment (AffectedArc07) [inline code/controllers/subsystem/redis.dm:96]: Feel like this should be something other than `log_game`.

Maybe `log_misc` or `log_world` or something.

**Comment by AffectedArc07** [comment]
PR bumped rustG redis from 0.21 to 0.22.3

That might help, though I am convinced its HW or something with this VM. Agh.

**Comment by AffectedArc07** [comment]
Todo - Revert the library change.

---

The real fix for this issue is to just not use linux lmao

**Comment by AffectedArc07** [comment]
Ok this is fully ready for merge now.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**